mmv1/third_party/terraform/resources/resource_sql_database_instance.go.erb

```diff
@@ -1561,6 +1568,7 @@ func flattenSettings(settings *sqladmin.Settings) []map[string]interface{} {
 		"activation_policy":          settings.ActivationPolicy,
 		"availability_type":          settings.AvailabilityType,
 		"collation":                  settings.Collation,
+		"connector_enforcement":      settings.ConnectorEnforcement,
```

Comment:
Here, if we check the sqladmin.settings, it uses 0.92v while the field connectorEnforcement was added in 0.93. Hence the field is not taken into consideration and build is failing with the error connectorEnforcement field not found. Any thoughts on this?

Reply:
I'm confused, what is the concern? If the sql version isn't a specific one the field may be omitted?

Reply:
That issue regarding older versions of sqladmin in use was fixed when it was upgraded from 0.92 to 0.97. But the current concerns are there are 2 tests failing for the resource but it doesn't seem like that is happening due to the field I've added. Even when run locally, Those tests are failing even when I use online build as well.